[operator-nexus] Update quickstarts-virtual-machine-deployment-bicep.md (#125079)
Azure/azure-content: articles/operator-nexus/quickstarts-virtual-machine-deployment-bicep.md

:::code language="bicep" source="includes/virtual-machine/virtual-machine-bicep-template.bicep":::

> [!IMPORTANT]

> > Please ensure that any sensitive data (e.g. secrets, passwords, private keys, etc.) are encrypted before they are submitted in the userdata or networkdata fields.

Reviewer: ```suggestion
> Please ensure that any sensitive data, such as secrets, passwords, private keys, and so on, are encrypted before they are submitted in the userData or networkData fields.
```
